packages/@react-aria/toast/intl/en-US.json

```diff
@@ -1,4 +1,5 @@
 {
   "close": "Close",
-  "notifications": "Notifications"
+  "notifications": "Notifications",
+  "countAnnouncement": "{count, plural, one {# notification} other {# notifications}}."
```

Comment:
I'm never seeing this announcement, when should it be happening?

Reply:
F6 to switch landmark regions is the easiest way.

Regarding your general comment it looks like the addition of focusing Toasts broke the focus management improvements. Waiting for feedback from Michael before fixing.

Reply:
I'm not seeing this announcement in the latest built version on Safari. @majornista how do I get this notification?